src/test/java/org/broadinstitute/hellbender/utils/runtime/StreamingProcessControllerUnitTest.java

```diff
@@ -97,7 +97,8 @@ public void testMultipleParallelStreamingControllers() throws TimeoutException {
         Assert.assertFalse(teeController.getProcess().isAlive());
     }
 
-    @Test(groups = "python", timeOut = 10000)
+    // The timeout value here needs to exceed the timeout value used by StreamingProcessControllerUnitTest.
```

Comment:
This comment should say "...used by StreamingProcessController", not "StreamingProcessControllerUnitTest.".